packages/calendar/src/Calendar.ts

```diff
@@ -0,0 +1,449 @@
+        return html`
+            <thead role="presentation">
+                <tr role="row">
+                    ${this._getWeekdays().map((weekday) =>
+                        this.renderWeekdayColumn(weekday)
+                    )}
+                </tr>
+            </thead>
+        `;
+    }
+
+    public renderWeekdayColumn(weekday: CalendarWeekday): TemplateResult {
+        return html`
+            <th
+                role="columnheader"
+                scope="col"
+                class="spectrum-Calendar-tableCell"
+            >
+                <abbr class="spectrum-Calendar-dayOfWeek" title=${weekday.long}>
+                    ${weekday.narrow}
+                </abbr>
+            </th>
+        `;
+    }
+
+    public renderCalendarTableBody(): TemplateResult {
+        return html`
+            <tbody role="presentation">
+                ${[...new Array(this._getWeeksInCurrentMonth()).keys()].map(
+                    (weekIndex) => this.renderCalendarTableRow(weekIndex)
+                )}
+            </tbody>
+        `;
+    }
+
+    public renderCalendarTableRow(weekIndex: number): TemplateResult {
+        return html`
+            <tr role="row">
+                ${this._getDatesInWeek(weekIndex).map((calendarDate) =>
+                    this.renderCalendarTableCell(calendarDate)
+                )}
+            </tr>
+        `;
+    }
+
+    public renderCalendarTableCell(calendarDate: CalendarDate): TemplateResult {
+        const isOutsideMonth = calendarDate.month !== this._currentDate.month;
+
+        const isSelected = Boolean(
+            this.selectedDate &&
+                isSameDay(this._toCalendarDate(this.selectedDate), calendarDate)
+        );
+
+        const isToday = Boolean(isSameDay(calendarDate, this.today));
+
+        const isDisabled = Boolean(
+            this.disabled ||
+                (this._minDate && calendarDate.compare(this._minDate) < 0) ||
+                (this._maxDate && calendarDate.compare(this._maxDate) > 0)
+        );
+
+        const dayClasses = {
+            'is-outsideMonth': isOutsideMonth,
+            'is-selected': isSelected,
+            'is-today': isToday,
+            'is-disabled': isDisabled,
+        };
+
+        const currentDayTitle = this._formatDate(calendarDate, {
+            weekday: 'long',
+            year: 'numeric',
+            month: 'long',
+            day: 'numeric',
+        });
+
+        // TODO: Translate "Today" and "selected"
+        const todayTitle = isToday ? 'Today, ' : '';
+        const selectedTitle = isToday ? ' selected' : '';
+
+        const title: string = this._capitalFirstLetter(
+            `${todayTitle}${currentDayTitle}${selectedTitle}`
+        );
+
+        return html`
+            <td
+                role="gridcell"
+                class="spectrum-Calendar-tableCell"
+                title=${title}
+                tabindex=${ifDefined(!isOutsideMonth ? '-1' : undefined)}
```

Comment:
undefined shouldn't be a fallback for tabIndex

Reply:
As I mentioned [earlier](https://github.com/adobe/spectrum-web-components/pull/3327#discussion_r1267816937), with `ifDefined` Lit will not render the attribute if the value is `undefined` 😁